src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -904,20 +981,34 @@ public void removeCitizen(CitizenData citizen)
     }
 
     /**
-     * Get citizen by ID
+     * Send the message of a removed workOrder to the client.
      *
-     * @param citizenId ID of the Citizen
-     * @return          CitizenData associated with the ID, or null if it was not found
+     * @param orderId the workOrder to remove.
+     */
+    public void removeWorkOrder(int orderId)
+    {
+        //  Inform Subscribers of removed workOrder
+        for (EntityPlayerMP player : subscribers)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'player' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)